samples/net/coaps_server/prj_qemu_x86.conf

```diff
@@ -14,7 +14,7 @@ CONFIG_NET_PKT_TX_COUNT=14
 CONFIG_NET_BUF_RX_COUNT=30
 CONFIG_NET_BUF_TX_COUNT=30
 CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=3
-CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=2
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
I'm actually more interested in discussing/settling on the general course of action - is it better to keep Z default config bare-bones, or make it more user-friendly and be able to work OOB with more apps (despite risks this approach also has)? If we agree it makes sense, then a user story can be submitted with specific criteria, and changes can be done en masse to implement to it (all this is not a priority, but can start to discuss/think now).